### DISCUSSION TITLE
Problem compiling on Linux.

### DISCUSSION THREAD

**Discussion by DavidChipman** [assimp/assimp#3932] [Q&A] [answered]
Hi all, I'm having trouble compiling Assimp on Fedora 34. I'm getting multiple errors about in the ZipArchiveIOSystem. The stderr output from g++ is in the attached file. Thanks for any help

[assimp.txt](https://github.com/assimp/assimp/files/6566075/assimp.txt)

**Comment by kimkulling** [comment]
Which version of g++ do you use? And have you generated the make-script by using cmake?

**Reply by DavidChipman** [reply]
I am using g++ ver. 11.1.1 and I have generated the Makefile using cmake.

**Comment by kimkulling** [comment]
If I remember right this issue was caused by a wrong zconf.h . Could you try our latest master and check if the build issue does stilll occurrs?

**Reply by DavidChipman** [reply]
Nope, still failing. This is an attempted build of assimp-master, not assimp-5.0.1 as I had tried before.

[assimp.txt](https://github.com/assimp/assimp/files/6639102/assimp.txt)

**Reply by kimkulling** [reply]
Hm, I need to check this.

**Comment by rodrygojose** [comment, marked as answer]
@DavidChipman did you manage to solve the issue? 
I had exactly the same problem. It is solved by compiling Assimp with its own `zlib`:
`cmake <path_to_CMakeLists.txt> -DASSIMP_BUILD_ZLIB=ON`
